javascript/ql/src/Security/CWE-020/IncompleteUrlSubstringSanitization.qll

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
Shouldn't this file be named `*Query.qll` and be moved into the `lib/` pack?

Reply:
It's not really a general purpose library and does not define any clear extension points. Is there a good reason to make this code part of the public API ?

Reply:
Hmmm, good point. Keep it.